packages/types/src/concordium/interfaces.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020-2023 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import {
+  AccountTransferredEvent,
+  BlockFinalizationSummary,
+  BlockInfo,
+  BlockItemSummary,
+  BlockSpecialEvent,
+  TransactionEvent,
+  TransactionSummaryType,
+  TransactionEventTag,
+} from '@concordium/node-sdk';
+
+export interface ConcordiumBlockFilter {
+  modulo?: number;
+  timestamp?: string;
+}
+
+export interface ConcordiumTransactionFilter {
+  type?: TransactionSummaryType;
+  values?: {
+    [key: string]: string;
+  };
```

Comment:
Are these not a known number of keys? Same question for the other filters

Reply:
keys are different for each type 
https://github.com/Concordium/concordium-node-sdk-js/blob/main/packages/common/src/types/blockItemSummary.ts